src/cli.ts

```diff
@@ -224,6 +224,10 @@ if (require.main === module) {
       '--internal-urls <value>',
       'regex of URLs to consider "internal"; all other URLs will be opened in an external browser. Default: URLs on same second-level domain as app',
     )
+    .option(
+      '--block-external-urls',
+      `disable navigation to URLs not considered "internal" (see '--internal-urls').  Instead of opening in an external browser, attempts to navigate to external URLs will be ignored. Default: false`,
```

Comment:
```suggestion
      `forbid navigation to URLs not considered "internal" (see '--internal-urls').  Instead of opening in an external browser, attempts to navigate to external URLs will be blocked. Default: false`,
```